components/_lists.scss

```diff
@@ -0,0 +1,19 @@
+.wds-list {
+	display: block;
+	list-style-type: none;
+	margin: 18px;
+	padding: 0;
+}
+
+.wds-list-item {
+	display: flex;
```

Comment:
Using flex for an item makes all tags in the item to be displayed as a separate flex item. Putting even an inline element such as `<a>` inside the list item will break the line.